src/main/java/edu/greenblitz/tobyDetermined/IsRobotReady.java

```diff
@@ -0,0 +1,44 @@
+package edu.greenblitz.tobyDetermined;
+
+
+import edu.greenblitz.tobyDetermined.commands.swerve.MoveToGrid.Grid;
+import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.ElbowSub;
+import edu.greenblitz.tobyDetermined.subsystems.telescopicArm.Extender;
+
+public class IsRobotReady {
+	public static boolean isArmAtGoal(){
+		switch (Grid.getInstance().getSelectedHeight()){
+			case HIGH:
+				if (Field.PlacementLocations.isGridPositionIDofCube(Grid.getInstance().getSelectedPositionID())){
```

Comment:
tal-function name is hard to read change id to index and use proper camelCase

Reply:
Agree